### ISSUE TITLE
catkin_python_setup() version in setup.py (0.3.0) differs from version in package.xml (0.3.1)

### ISSUE THREAD

**Issue by sloretz** [Achllle/dual_quaternions#2]
http://build.ros.org/job/Nbin_uF64__dual_quaternions__ubuntu_focal_amd64__binary/1/console

```
12:16:20 CMake Error at /opt/ros/noetic/share/catkin/cmake/catkin_python_setup.cmake:56 (message):
12:16:20   catkin_python_setup() version in setup.py (0.3.0) differs from version in
12:16:20   package.xml (0.3.1)
12:16:20 Call Stack (most recent call first):
12:16:20   CMakeLists.txt:8 (catkin_python_setup)
```

Looks like there's an issue with the `0.3.1` tag 

https://github.com/Achllle/dual_quaternions/blob/19bee60b31a151eaf428d67da3fedda85097b6e5/setup.py#L8

**Comment by Achllle** [comment]
Huh `master` has `0.3.1`. Simply `bloom-release` again to remedy? @sloretz

**Comment by sloretz** [comment]
`master` looks ok, but the `0.3.1` tag does not.

https://github.com/Achllle/dual_quaternions/tree/0.3.1

If I understand correctly running `bloom-release` with no changes will just re-release the same `0.3.1` tag. I think the easiest thing to do would be to make a new ` 0.3.2` tag and then `bloom-release` that

**Comment by Achllle** [comment]
How about removing the `0.3.1` tag and recreating it with the commit?

**Comment by sloretz** [comment]
> How about removing the 0.3.1 tag and recreating it with the commit?

I think that would work too, though changing existing tags can be inconvenient for downstream users. If someone has cloned your repo and pulled the existing `0.3.1` tag then they will need to run `git pull --tags --force` locally to get the updated one.

**Comment by Achllle** [comment]
This repo isn't popular enough for that (yet) :)

**Comment by Achllle** [comment]
@sloretz Question: from the [latest build](http://build.ros.org/job/Nbin_ufv8_uFv8__dual_quaternions__ubuntu_focal_arm64__binary/15/console) it seems like this strategy didn't work. When I create a new tag, do I need to update releases for kinetic and melodic? If not, do I just skip a version next time for those platforms?

**Comment by Achllle** [comment]
I released 0.3.2 since just force-pushing 0.3.1 had no effect on the debian packages that bloom-release generates...
